elasticdl/python/common/args.py

```diff
@@ -289,6 +289,9 @@ def add_common_params(parser):
         'by semi-colon used to debug , e.g. "param1=1; param2=2" '
         "Supported auxiliary parameters: disable_relaunch",
     )
+    parser.add_argument(
+        "--log_file_path", type=str, default="", help="The path to save logs",
```

Comment:
What type of logs? e.g. stdout, stderr

Reply:
The stdout and stderr logs.  I have added the comment in the help.